README.md

```diff
@@ -55,15 +55,21 @@ Using the Hotswap Agent is recommended if you want to do live code editing. Plea
 but bear in mind that the instructions are incorrect (!). Don't "unpack" `hotswap-agent.jar`, instead
 rename the downloaded jar file to `hotswap-agent.jar`. Finally, add `wynntils.hotswap=true` in your personal `gradle.properties` file.
 By default, this is `C:\Users\<your username>\.gradle\gradle.properties` on Windows, or `~/.gradle/gradle.properties` on Linux/MacOS.
+If the file does not exist, you can create it yourself.
 Don't forget to set the correct Java installation for your run configurations, and make sure to use [these](https://i.imgur.com/4VMFCM0.png) settings in IntelliJ IDEA.
 
+#### Windows
+If you are using Hotswap Agent on Windows and experience the "The process cannot access the file because it is being used by another process" error, you can replace your Architectury transformer runtime as a temporary fix.
```

Comment:
The formatting is a bit odd. If you intend this as a continuous paragraph (which it will be when rendered as html), please write so in markdown as well. If you want separate paragraphs you need an empty line between.